src/vmware/azext_vmware/_help.py

```diff
@@ -298,6 +298,19 @@
       text: az vmware datastore disk-pool-volume create --name iSCSIDatastore1 --resource-group MyResourceGroup --cluster Cluster-1 --private-cloud MyPrivateCloud --target-id /subscriptions/11111111-1111-1111-1111-111111111111/resourceGroups/ResourceGroup1/providers/Microsoft.StoragePool/diskPools/mpio-diskpool/iscsiTargets/mpio-iscsi-target --lun-name lun0
 """
 
+helps['vmware datastore elastic-san-volume'] = """
+    type: group
+    short-summary: Create an Elastic SAN volume in a private cloud cluster using Microsoft.ElasticSan provider.

```

Comment:
It is a command group, please check this short summary.
```suggestion
    short-summary: Manage Elastic SAN volume resource.

```

Reply:
Fixed and updated similar issues.